docs/src/userguide/wrapping_CPlusPlus.rst

```diff
@@ -667,17 +667,23 @@ in ``libcpp.typeindex``.
 Specify C++ language in setup.py
 ================================
 
-Instead of specifying the language and the sources in the source files, it is
-possible to declare them in the :file:`setup.py` file::
+Instead of specifying the language in the source file (using a ``distutils``
+directive), it is possible to declare it in the :file:`setup.py` file.
+This is done by creating an :class:`~setuptools.Extension` object for your
+extension::
```

Comment:
```suggestion
Instead of specifying the language in the source file (using a ``distutils``
directive, see below), it is possible to declare it in the :file:`setup.py` file.
This is done by creating an :class:`~setuptools.Extension` object for your
extension::
```